charmcraft.yaml

```diff
@@ -2,14 +2,15 @@ type: charm
 bases:
   - build-on:
       - name: ubuntu
-        channel: "22.04"
+        channel: "20.04"
     run-on:
       - name: ubuntu
         channel: "20.04"
         architectures: [amd64]
+  - build-on:
       - name: ubuntu
-        channel: "21.10"
-        architectures: [amd64]
+        channel: "22.04"
+    run-on:
       - name: ubuntu
         channel: "22.04"
         architectures: [amd64]
```

Comment:
Unfortunately substrate-interface package need the charm to be built on the same series that it will run on. When building on Jammy and run on Focal it could not import xxhash which is a dependency for substrate-interface.

I also removed support for 21.10. It's kind of a weird release to support I would say. It's also end of life.

Reply:
Sounds good, the need for 21.10 is probably negligible anyway!